submissions/krylenger/friends/app.js

```diff
@@ -141,86 +147,85 @@ const filterCards = (usersData) => {
   }
-  if (filtersState.sortRegistrationDate === 'registrationDate_OLD') {
-    filteredUsersData = sortByParam(filteredUsersData, 'registrationDate', 'descending');
-  } 
 
   renderUserCards(filteredUsersData);
-}
+};
 
 const addEventListeners = (usersData) => {
-
-  search.addEventListener('keyup', ({target}) => {
+  search.addEventListener("keyup", ({ target }) => {
     filtersState.search = target.value;
     filterCards(usersData);
-  })
+  });
 
   asideFilterGenderContainer.addEventListener("click", ({ target }) => {
     filtersState.gender = target.id;
-		filterCards(usersData);
+    filterCards(usersData);
   });
 
   asideFilterCountry.addEventListener("change", ({ target }) => {
-		filtersState.country = target.value;
-		filterCards(usersData);
+    filtersState.country = target.value;
+    filterCards(usersData);
   });
 
   asideFilterAge.addEventListener("input", ({ target }) => {
     asideRangeAgeOutput.textContent = target.value;
-		filtersState.age = target.value;
-		filterCards(usersData);
+    filtersState.age = target.value;
+    filterCards(usersData);
   });
 
   buttonSortContainerName.addEventListener("click", ({ target }) => {
-		filtersState.sortName = target.id;
-		filtersState.sortAge = null;
-		filtersState.sortRegistrationDate = null;
-		filterCards(usersData);
+    filtersState.sortName = target.id;
+    filtersState.sortAge, filtersState.sortRegistrationDate = null;
+    filterCards(usersData);
   });
 
   buttonSortContainerAge.addEventListener("click", ({ target }) => {
-		filtersState.sortAge = target.id;
-		filtersState.sortName = null;
-		filtersState.sortRegistrationDate = null;
-		filterCards(usersData);
+    filtersState.sortAge = target.id;
+    filtersState.sortName, filtersState.sortRegistrationDate = null;
+    filterCards(usersData);
   });
 
-  buttonSortContainerRegistrationDate.addEventListener("click", ({ target }) => {
-		filtersState.sortRegistrationDate = target.id;
-		filtersState.sortAge = null;
-		filtersState.sortName = null;
-		filterCards(usersData);
-  });
+  buttonSortContainerRegistrationDate.addEventListener(
+    "click",
+    ({ target }) => {
+      filtersState.sortRegistrationDate = target.id;
+      filtersState.sortAge, filtersState.sortName = null;
+      filterCards(usersData);
+    }
+  );
 
   buttonReset.addEventListener("click", () => {
-		filtersState.sortName = null;
-		filtersState.sortAge = null;
-    filtersState.sortRegistrationDate = null;
-    filtersState.search = null;
-    filtersState.age = null;
-    filtersState.gender = null;
-    filtersState.countries = null;
-    search.value = '';
-		renderUserCards(usersData);
+    filtersState.sortName,
+      filtersState.sortAge,
+      filtersState.sortRegistrationDate,
+      filtersState.search,
+      filtersState.age,
+      filtersState.gender,
+      filtersState.countries = null;
+    search.value = "";
+    renderUserCards(usersData);
```

Comment:
Its the same, what will you do if you have 100 fields in the object?
hint: `Object.keys()` `Array.prototype.forEach`

Reply:
Thanks, got the idea! Really helps to make code smaller.
I did it using for in loop 
✅ Done